docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -1,21 +1,37 @@
 import * as React from 'react'
 import Scrollbars from 'react-custom-scrollbars'
-import { Chat, Divider } from '@stardust-ui/react'
-
+import { Chat, Divider, ChatMessage } from '@stardust-ui/react'
 import { ChatData, ChatItemTypes, generateChatProps } from './services'
 
+const ariaLiveStyle: React.CSSProperties = {
+  border: '0px',
+  clip: 'rect(0px, 0px, 0px, 0px)',
+  height: '1px',
+  margin: '-1px',
+  overflow: 'hidden',
+  padding: '0px',
+  width: '1px',
+  position: 'absolute',
+}
+
 export interface ChatPaneContainerProps {
   chat: ChatData
 }
 
 class ChatPaneContainer extends React.PureComponent<ChatPaneContainerProps> {
   public render() {
-    const items = this.generateChatItems(this.props.chat)
+    const chat = this.props.chat
```

Comment:
nit: `props` object could be decomposed here, just to be consistent with other places in code:

```js
const { chat } = this.props
```